### ISSUE TITLE
Website hijacked?

### ISSUE THREAD

**Issue by richarddewit** [redstone-dart/redstone.dart#200]
Random links to pharmaceuticals are inserted:

![image](https://user-images.githubusercontent.com/7933750/80180226-8c4af280-8602-11ea-8e75-6b83aab922f7.png)

Also Wappalyzer states that it's built with PHP instead of Github Pages

**Comment by luizmineo** [comment]
Yeah, it's my fault, I forgot to renew the redstone.dart domain. I updated the repository description to link directly to gh-pages, and sent an abuse report to the domain registrar, but I don't think we can get the domain back.

Note that this package is unmaintained for a long time, and it's not compatible with dart 2.0, so I don't recommend to rely on it. I will update the README and the website with this information later.
